hooks/queries/user/useDesignatedVotingV1Address.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from "@tanstack/react-query";
+import { getDesignatedVotingV1Address } from "helpers";
+
+export function useDesignatedVotingV1Address(address: string) {
+  return useQuery(
+    ["designatedVotingV1Address", address],
+    () => getDesignatedVotingV1Address(address),
+    {
+      enabled: !!address,
```

Comment:
```suggestion
      enabled: !!address && !isWrongChain,
```

we should add an `isWrongChain` check so that this doesn't throw on the wrong chain